src/styles.scss

```
@@ -1,4 +1,11 @@
 /* You can add global styles to this file, and also import other style files */
 
-html, body { height: 100%; overflow-y: hidden;}
-body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+html, body { 
+height: 100%; 
```

```suggestion
     height: 100%; 
```
Coloca tabulacao aqui ... vou acrescentar um linter no codigo ... 
os atributos de dentro de chaves, sempre bom estar com um espacamento da borda.